checkov/arm/checks/resource/AppServiceRemoteDebuggingNotEnabled.py

```diff
@@ -0,0 +1,20 @@
+from checkov.common.models.enums import CheckCategories
+from checkov.arm.base_resource_value_check import BaseResourceValueCheck
+
+
+class AppServiceRemoteDebuggingNotEnabled(BaseResourceValueCheck):
+    def __init__(self) -> None:
+        name = "Ensure that remote debugging is not enabled for app services"
+        id = "CKV_AZURE_72"
+        supported_resources = ('Microsoft.Web/sites',)
+        categories = (CheckCategories.GENERAL_SECURITY,)
+        super().__init__(name=name, id=id, categories=categories, supported_resources=supported_resources,)
```

Comment:
Please add `missing_block_result=CheckResult.PASSED` and an example of a resource without `remoteDebuggingEnabled` field.